### PR TITLE
dev-java/tomcat-native: Modified to build jar

### PR DESCRIPTION
**Pull request by wltjr**
@gentoo/java Modified ebuild to build the jar in addition to the .so

### CODE REVIEW

File: dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild [hunk 1]
```diff
@@ -13,30 +13,42 @@ SRC_URI="mirror://apache/tomcat/tomcat-connectors/native/${PV}/source/${P}-src.t
...
 src_configure(){
+	cd native
 	econf --with-apr=/usr/bin/apr-1-config  \
 		--with-ssl=/usr || die "Could not configure native sources"
 }
 
 src_compile() {
+	eant jar -f build.xml
+
+	cd native
```
Comment (hasufell) [inline dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild]: missing `|| die`

File: dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild [hunk 2]
```diff
@@ -13,30 +13,42 @@ SRC_URI="mirror://apache/tomcat/tomcat-connectors/native/${PV}/source/${P}-src.t
...
 	base_src_compile
 }
 
 src_install() {
+	java-pkg_newjar dist/${PN}-${PV}-dev.jar ${PN}.jar
+
+	cd native
```
Comment (hasufell) [inline dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild]: missing `|| die`

**Comment by hasufell** [comment]
@monsieurp @chewi

File: dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild (hunk 2 above)
Comment (monsieurp) [inline dev-java/tomcat-native/tomcat-native-1.1.33-r1.ebuild]: https://github.com/gentoo/gentoo/commit/6d93b2e2efe4034288ceb9f61f932bef44cb1af1 addresses your remarks. Thank you!

**Comment by monsieurp** [comment]
Merged! :+1:

**Pull request closed without merging**